ray-curator/ray_curator/models/pos_embed.py

```diff
@@ -0,0 +1,326 @@
+# Copyright (c) OpenGVLab. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+from typing import Any
+
+import numpy as np
+import torch
+
+logger = logging.getLogger(__name__)
```

Comment:
from loguru import logger

can you please ensure this is consistent across the video pipeline. Please make a ticket for this to track? For now we can merge this but let's make sure the consistency.